styles.css

```diff
@@ -0,0 +1,411 @@
+  height: 250px;
+}
+
+.inmag-bg {
+  background-color: #f51e01;
+  margin-top: 3px;
+}
+
+.t-red {
+  color: #f51e01;
+}
+
+.nw-icon {
+  width: 34px;
+  height: auto;
+}
+
+.division {
+  width: 100vw;
+}
+
+.subscription-sec,
+.subscription-sec-b {
+  display: flex;
+  flex-direction: row;
+  background-color: #f4f5f6;
+  height: 350px;
+  overflow: hidden;
+}
+
+.sub-sec-right {
+  display: flex;
+  flex-direction: column;
+}
+
+.news-sec-right {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.news-sec-img {
+  width: 40%;
+}
+
+.sub-sec-right p {
+  color: #62696b;
+}
+
+.sub-btn {
+  font-weight: 500;
+  font-size: 18px;
+}
+
+.news-week-footer-logo {
+  width: 280px;
+}
+
+.social-icons ul li {
+  position: relative;
+  top: 26px;
+  display: inline;
+  margin-right: 10px;
+  color: #62696b;
+}
+
+.editions {
+  position: relative;
+  right: 38px;
+  bottom: 15px;
+}
+
+.editions-b {
+  position: relative;
+  right: 38px;
+  bottom: 30px;
+}
+
+.editions ul li {
+  display: inline;
+  margin-right: 10px;
+  color: #62696b;
+  font-size: 14px;
+}
+
+.copyright {
+  position: relative;
+  top: 17px;
+}
+
+.line {
+  color: #7f7f7f;
+}
+
+.footer-top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.cursor-p {
+  cursor: pointer;
+}
+
+.hv-hd:hover {
+  text-decoration: underline;
+  text-decoration-color: #f51e01;
+}
+
+.hv-hd-a:hover {
+  color: #f51e01;
+}
+
+.na:hover {
+  text-decoration: none;
+  cursor: none;
+}
+
+.social-icons,
+.editions ul li:hover,
+.editions-b ul li:hover {
+  cursor: pointer;
+  color: #000;
+}
+
+/* Media Queries */
+
+@media (max-width: 990px) {
+  main {
+    margin-top: 60px;
+  }
+
+  .nav-logo {
+    display: none;
+  }
+
+  .nav-item-margin {
+    position: relative;
+    top: 0;
+  }
+
+  .latest-new-sec {
+    width: 35%;
+  }
+
+  .scroll-right {
+    height: initial;
+    overflow-y: hidden;
+  }
+
+  .sub-sec-right {
+    width: 100%;
+  }
+
+  .subscription-sec-b {
+    height: 200px;
+  }
+
+  .news-sec-right {
+    width: 75%;
+  }
+
+  .news-sec-img {
+    width: 25%;
+  }
+
+  .sub-sec-mag {
+    width: 40%;
+  }
+}
+
+@media (max-width: 770px) {
+  .top-story {
+    order: 1;
+  }
+
+  .feature-story {
+    order: 2;
+  }
+
+  .opinion-story {
+    order: 3;
+  }
+
+  .latest-new-sec {
+    width: 100%;
+  }
+
+  .culture-n-travel {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .ct-article {
+    width: 100%;
+  }
+
+  .scroll {
+    height: initial;
+    overflow-y: hidden;
+  }
+
+  .inmag-h {
+    height: auto;
+  }
+
+  .subscription-sec {
+    display: flex;
+    flex-direction: column;
+    background-color: #f4f5f6;
+    height: 520px;
+    text-align: center;
+  }
+
+  .sub-sec-mag {
+    width: 40%;
+    text-align: center;
+  }
+
+  .line {
+    display: none;
+  }
+
+  .footer-top {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .footer-top-left {
+    display: flex;
+    flex-direction: column;
```

Comment:
Replace with bootstrap classes

Reply:
This was created to avoid breaking of the way footer was designed, use of bootstrap breaks the layout drastically